mrp_multi_level/views/mrp_planned_order_views.xml

```diff
@@ -0,0 +1,135 @@
+                    </div>
+                    <group>
+                        <group>
+                            <field name="product_mrp_area_id" />
+                            <field name="product_id" />
+                            <field name="mrp_area_id" />
+                            <field name="name" />
+                            <field name="fixed" />
+                        </group>
+                        <group>
+                            <field name="mrp_action" />
+                            <field name="order_release_date" />
+                            <field name="due_date" />
+                            <field name="mrp_qty" />
+                            <field name="qty_released" />
+                        </group>
+                    </group>
+                    <group groups="base.group_no_one">
+                        <field name="mrp_move_down_ids" readonly="True" />
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record id="mrp_planned_order_view_pivot" model="ir.ui.view">
+        <field name="name">mrp.planned.order.pivot</field>
+        <field name="model">mrp.planned.order</field>
+        <field name="arch" type="xml">
+            <pivot>
+                <field name="mrp_area_id" type="row" />
+                <field name="product_id" type="row" />
+                <field name="mrp_qty" type="measure" />
+                <field name="due_date" interval="week" type="col" />
+            </pivot>
+        </field>
+    </record>
+    <record id="mrp_planned_order_view_search" model="ir.ui.view">
+        <field name="name">mrp.planned.order.search</field>
+        <field name="model">mrp.planned.order</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="product_id" />
+                <field name="mrp_area_id" />
+                <separator />
+                <filter string="Fixed" name="fixed" domain="[('fixed','=',True)]" />
+                <group name='group_by' expand="0" string="Group By...">
+                    <filter
+                        name='product_parameters'
+                        string="Product Parameters"
+                        context="{'group_by':'product_mrp_area_id'}"
+                    />
+                    <filter
+                        name='due_date_by_day'
+                        string="Due Date (By Day)"
+                        context="{'group_by':'due_date:day'}"
+                    />
+                    <filter
+                        name='due_date_by_week'
+                        string="Due Date (By Week)"
+                        context="{'group_by':'due_date:week'}"
+                    />
+                    <filter
+                        name='due_date_by_month'
+                        string="Due Date (By Month)"
+                        context="{'group_by':'due_date:month'}"
+                    />
```

Comment:
keep only one filter, and remove what it is in parenteris. In v13 you can select the date range by default in this type of filters.